packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -8,7 +8,7 @@
 
 Summary: Katello command plugin for the Hammer CLI
 Name:    %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.13.2
+Version: 0.14.0
```

Comment:
The problem with this is that we can't test changes. The current releaser can't find the actual gem. Changing the releaser to nightly means we can't actually release. I still think that we should keep this package spec + releaser to production settings and explicitly set the releaser in the nightly job until obal learns how to build nightly packages.

Reply:
Would you mind explaining more what is preventing us from testing changes or doing an "actual release?" It's not that I want to argue, but I am having trouble understanding what mechanisms wouldn't work.

Reply:
When we release or test (scratch) this releaser is used. It means that if we would want to release an actual `0.14.0` then you need to override the releaser. In a PR that can't happen unless you change `package_manifest.yaml`. Currently the hammer (plugin) maintainer sends in PRs to update the version. These 2 workflows are incompatible.

We either need to change it so the release manager includes hammer updates when doing a Foreman/Katello release or keep the current workflow.

Reply:
When a PR is made to this repo (foreman-packaging) that kicks off the [foreman-packaging-rpm-pr-test](http://ci.theforeman.org/job/foreman-packaging-rpm-pr-test/) which `obal scratch` builds packages using what is listed in the package manifest.

The changes introduced in theforeman/foreman-infra#708 are for a different jenkins job ([hammer-cli-katello-master-release](http://ci.theforeman.org/job/hammer-cli-katello-master-release)) which clones the project repo, builds a gem locally, checks out f-packaging and submits an `obal release` using custom releasers and extra vars.

It sounds like these two workflows won't interfere with one another?

Reply:
Actually this was already a problem when I tried to `obal release` 54d77e1a17d076b3be63e97119bc2a0ca7b1531f. I built a nightly package twice before I realized that it doesn't work that way. That's my main problem. As I understand it now:

For foreman and foreman-{installer,proxy,selinux} we never check in the source in nightly. That's only done in a branch. That's no problem because such a release (1.18.0) by definition only ever lives in a branch (1.18-stable).

For hammer we check in the source gem and release from nightly. Then we cherry pick to older releases as needed. As the linked commit: it's supposed to live in 1.18, 1.19 and nightly. Because there is no dist release we must tag it in koji.

I don't know what's the best workflow here. Should we add a dist tag and just build it once for every branch? Then we should remember that after branching we explicitly need to include a hammer release where we now inherit it automatically from nightly.